wuvt/defaults.py

```diff
@@ -28,8 +28,8 @@
 ]
 
 # DJ activity timer for automation/DJ logout in seconds
-DJ_TIMEOUT = 30 * 60
-EXTENDED_DJ_TIMEOUT = 120 * 60
+DJ_TIMEOUT = 60 * 60
```

Comment:
I think this should be 90 minutes; since this is the shortest show length and should exceed the longest possible tracks (I'm thinking of the 5 hour radiothon show in which I played 3 pieces, total, as an exception), this is the longest a DJ should be able to not log tracks. There are just enough tracks that are 65 to 75 minutes, I think.